packages/grid/data-grid/src/DataGrid.tsx

```diff
@@ -1,7 +1,8 @@
 import * as React from 'react';
 import * as PropTypes from 'prop-types';
-import { chainPropTypes } from '@material-ui/utils';
-import { GridComponent, GridComponentProps, classnames } from '@material-ui/x-grid-modules';
+import { GridComponent, GridComponentProps, classnames } from '../../_modules_/grid';
+
+const chainPropTypes = require('@material-ui/utils').chainPropTypes;
```

Comment:
```suggestion
import { chainPropTypes } from '@material-ui/utils';
```

Reply:
this create issues, I suspect it's because there is no d.ts files 

> > Where did you run this command from?
> 
> 1. Open codesandbox-ci, get the dependency URLs
> 2. Use the dependencies URLs in the issue reproduction (see the pull request's description)
> 3. yarn
> 4. yarn typescript
> 
> 💥

test with data-grid, and no usage of data gen for now

Reply:
I tested locally and it worked

Reply:
Ok, I guess we can wait. Should we add a TODO v5 comment?